Security-Team.md

```diff
@@ -106,3 +106,60 @@ the Node.js program on HackerOne.
 - [@watson](https://github.com/watson) - Thomas Watson
 
 <!-- ncu-team-sync end -->
+
+## Security Triage Workflow
+
+Security issues are reported via HackerOne, see [nodejs.org](https://nodejs.org/en/security/#reporting-a-bug-in-node-js).
+
+Report states are described in HackerOne [docs](https://docs.hackerone.com/programs/report-states.html).
+
+The expected workflow for issues reported to Node.js is:
+
+1. `NEW`: issue reported (initial state).  Issues in this state are in the
+   process of being analyzed with the intention to move them into either
+   `TRIAGED`, `NEEDS-MORE-INFO`, or one of the "Closed" states
+   (`NOT-APPLICABLE`, `INFORMATIVE`, etc.). Has several sub-states:
+   1. `H1 Triage`: Issues start off assigned to the `H1 Triage` team. That team
+      does initial triage. They ask for a runnable reproduction, they run it,
+      and they verify whether it does in fact appear to demonstrate a
+      vulnerability that has not already been reported. Once this is done, they
+      assign the issue to the `Node.js Team`.
+   2. `Node.js Team`: Issues assigned by `H1 Triage`, or issues that Node.js
+      assigns to itself for triage preemptively (perhaps because of priority, or
+      because its disposition is obvious, or some other reason we decide not to
+      skip the `H1 Team`'s reproduction).
+   3. `NEEDS-MORE-INFO`: In the state of back-and-forth with reporter.
+      When further progress on the issue is blocked on response from the reporter,
+      this is the state.
+2. `TRIAGED`: State for an issue when it is a non-duplicate and resolvable.
+   Issues in this state are waiting for the Node.js team to take some action to
+   resolve them. Typically, this would be publishing a Node.js release that
+   resolves the issue, but it could also be publishing documentation, or making
+   a configuration change to our infrastructure for a https://nodejs.org
+   problem. What the resolution should be should be described in a comment on
+   the issue when it is moved to `TRIAGED`.
+   - ...: There is no explicit states for an issue as it is in process of
+     getting resolved. Comments on the issue when someone is working on it are
+     useful, as is assignment of the issue to a person who agress to be
+     responsible for making sure it is resolved.
+5. `RESOLVED`: State for an issue that has had a fix published. Issues in this
+   state should be disclosed.
+6. ...: Final states for issues that we will not fix:
+   - `NOT-APPLICABLE`: We do not agree this a vulnerability. We can request
+     that the reporter close it to preserve their H1 "reputation points".  If
+     its an issue, but not a vulnerability, we can suggest that it be reported
+     to the Node.js issue tracker. ***Should it be disclosed?***
```

Comment:
Are these ***Should it be disclosed*** questions?

Reply:
yes

Reply:
Story still a bit confused.  So this is a WIP versus a doc ready to land?

Reply:
these lines don't exist anymore